### PR TITLE
Added new Sublime theme

### PR DESCRIPTION
**Pull request by mortalis13**

### CODE REVIEW

**Comment by nightwing** [comment]
To merge this i have to ask you to send a cla to https://github.com/ajaxorg/ace/blob/master/CONTRIBUTING.md.

Btw is this based on an existing theme for another editor? Sublime name is a bit confusing, I expected this to be something related to sublime text.

File: lib/ace/theme/sublime.css [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  color: #333333
+}
+
+.ace-sublime .ace_cursor {
+  color: #333
+}
+
+.ace-sublime .ace_marker-layer .ace_selection {
+  background: #BAD6FD;
+  border-radius: 4px
```
Comment (nightwing) [inline lib/ace/theme/sublime.css]: This is wrong it adds round corners in the middle of the selection.

File: lib/ace/theme/sublime.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  color: #a55f03
+}
+
+.ace-sublime .ace_comment {
+  color: #777777;
+  font-style: italic
+}
+
+.ace-sublime .ace_fold-widget {
+  background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAANElEQVR42mWKsQ0AMAzC8ixLlrzQjzmBiEjp0A6WwBCSPgKAXoLkqSot7nN3yMwR7pZ32NzpKkVoDBUxKAAAAABJRU5ErkJggg==);
```
Comment (nightwing) [inline lib/ace/theme/sublime.css]: Are black indent guides and print margin intentional? I wold expect them to be something lighter.

**Comment by mortalis13** [comment]
1. The theme is related to the Sublime Text. Of course it's not its default theme, but the theme I use now. I edited it too for my needs. So I copied the styles from it.

2. Do you want me to remove the radius and lighten indents and margin?

**Comment by nightwing** [comment]
1. What Is it called in sublime text? i cannot find it on package control.
2. yes, please do, i think with lighter indents and margin it will look much better.

**Comment by mortalis13** [comment]
1. It's derivative from `iPlasctic`.
2. No problem.

**Comment by mortalis13** [comment]
I'm relatevely new to Git. Have I done all I had to?
Or I should create another pull request?

**Comment by nightwing** [comment]
1. then we need to call it something based on iPlastic, sublime is misleading.
2. looks good now, you do not need to create a new pull request. Rebasing on master and merging commits to not have several commits with `edited sublime` message, would be nice, but is not absolutely necessary.

**Comment by mortalis13** [comment]
Let it be `iPlastic` then. I didn't change much in the original theme.

**Comment by nightwing** [comment]
Thanks, merged.

**Comment by mortalis13** [comment]
Thank you for your support.  
Hope I'll add something more useful that a theme in the future...

**Pull request merged**